isis/tests/FunctionalTestsIsis2Std.cpp

```diff
@@ -0,0 +1,1054 @@
+#include "gmock/gmock.h"
+
+#include <memory>
+
+#include <QFile>
+#include <QString>
+#include <QVector>
+
+#include "isis2std.h"
+
+#include "Cube.h"
+#include "std2isis.h"
+#include "Fixtures.h"
+#include "Histogram.h"
+#include "LineManager.h"
+#include "Portal.h"
+#include "SpecialPixel.h"
+#include "Statistics.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/isis2std.xml").expanded();
+static QString STD2ISIS_XML = FileName("$ISISROOT/bin/xml/std2isis.xml").expanded();
+
+class IsisTruthCube : public TempTestingFiles {
```

Comment:
What's the rationale for defining the fixtures here instead of fixtures.cpp?

Reply:
No real reason. It's faster to not add stuff to that as it causes re-compiling all of the tests whenever the header changes. If these could be useful elsewhere then I can move them.